pylint_odoo/test/main.py

```diff
@@ -230,6 +231,14 @@ def test_80_ignore_patternls(self):
         self.assertEqual(real_errors.items(),
                          [('deprecated-openerp-xml-node', 3)])
 
+    def test_90_xml_attribute_translatable(self):
+        extra_params = ['--disable=all',
+                        '--enable=xml-attribute-translatable']
+        pylint_res = self.run_pylint(self.paths_modules, extra_params)
```

Comment:
One more time! Do you have reason!
I had included it because the test_25_checks_without_coverage method asked me to have a test, but it is no longer necessary!